tests/test_search.py

```diff
@@ -183,3 +183,37 @@ def test_get_swath_polygon():
 def test_swath_polygon_not_found():
     with pytest.raises(ValueError, match="No swath polygons found for workunit="):
         coincident.search.wesm.get_swath_polygons("AL_SWCentral_1_B22")
+
+
+# opentopo (NCALM and NOAA)
+# =======
+# TODO: remove datetime argument constraint for opentopo search
+# also, smaller aois
+@network
+def test_noaa_search(bathy_aoi):
+    gf = coincident.search.search(
+        dataset="noaa", intersects=bathy_aoi, datetime=["2019-01-01", "2023-12-31"]
+    )
+    assert len(gf) == 2
```

Comment:
I suggest adding a couple more assertions, for example, you can check that we're getting certain column names:

https://github.com/uw-cryo/coincident/blob/e7a778f653c1ff1c6e2177b8bf87555880d8f4a4/tests/test_search.py#L89-L90

It occurs to me that especially with the external API results we can be more specific to ensure we're getting what we expect in the future, e.g. `assert gf.shape == (10,2)` ... I'll plan to update other tests to do this. This way we'll catch updates from the data providers if they change field names or add new ones to their outputs.